tools/slurm_test.sh

```diff
@@ -12,6 +12,7 @@ CPUS_PER_TASK=${CPUS_PER_TASK:-5}
 PY_ARGS=${@:5}
 SRUN_ARGS=${SRUN_ARGS:-""}
 
+PYTHONPATH=$(cd "$(dirname $0)/..";pwd):$PYTHONPATH \
```

Comment:
We may extract `PYTHONPATH=$(cd "$(dirname $0)/..";pwd):$PYTHONPATH` above and pass `$PYTHONPATH` here.